backend/compact-connect/docs/it_staff_onboarding_instructions.md

```diff
@@ -237,9 +241,49 @@ For your convenience, use of this feature is included in the [Postman Collection
 - All date fields must use the `YYYY-MM-DD` format
 - The API does not accept `null` values. For optional fields with no value, omit the field or leave it empty in CSV.
 
+## Recommended Upload Strategies: JSON vs CSV
```

Comment:
This is just a personal nit so feel free to disregard if you don't agree: I'm not sure I'd go so far as calling these all _recommendations_. What we're describing are common strategies that states have successfully adopted. I think calling them our _recommendations_ are is a bit strong, given that, for the JSON endpoint, we're describing using an endpoint that supports efficient processing of 100 records to only process one at a time. I'd personally lean towards calling this 'common strategies' and describe both a dead-simple-one-at-a-time strategy and a more efficient 100-at-a-time strategy that involves removing identified invalid records. The former has the advantage of simplicity, while the latter has the advantage of gracefully supporting higher throughput within the API rate limit.